app/controllers/authorized_family_members_controller.rb

```diff
@@ -34,7 +34,7 @@ def update
   private
 
   def family
-    @_family ||= current_partner.families.find_by(id: params[:family_id])
+    @_family ||= current_partner.families.find_by(params[:family_id])
```

Comment:
I think we need `find` instead of `find_by` for this to work?

Reply:
Well I'm not sure why, but that was the way I made it work. I tried this:
`@_family ||= current_partner.families.find_by(family_id: params[:family_id])`
`@_family ||= current_partner.families.find_by(id: params[:family_id])`
`@_family ||= current_partner.families.find(params[:family_id])` all failed.
The request returns a 500(internal_server_error).

Then I put a binding.pry, to check this params[:family_id], thinking that it was a hash. But it was: `[1] pry(#<AuthorizedFamilyMembersController>)> params[:family_id]
=> nil`

Reply:
@m-pereira it was pretty odd case you found here. I didn't realize that it seems that `.find_by(nil)` returns the last record. Thus is it will pass because a `Family` record is found. However, it won't be testing what you want to be testing.

The test is failing because of the `params[:family_id]` is not being defined in your spec or the way you are calling the endpoint. I'll mention more in the spec.